### ISSUE TITLE
`helm template` does not have -x flag

### ISSUE THREAD

**Issue by waveywaves** [helm/helm#6305]
Upon running `helm template traefik/ -x templates/deployment.yaml`. I get the following output
`Error: unknown shorthand flag: 'x' in -x`

Output of `helm version`:

version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"27e70c8f1da5be31a27512b97dc4bda2a9b7c90a", GitTreeState:"clean", GoVersion:"go1.12.6"}
Output of `kubectl version`:

Cloud Provider/Platform (AKS, GKE, Minikube etc.):

**Comment by waveywaves** [comment]
Seems that `helm template` wasn't properly ported over from helmv2. It makes sense to have to for templating individual resources.

**Comment by hickeyma** [comment]
Closing as duplicate of #6290

**Comment by hickeyma** [comment]
Better to say that this issue and other flags missing are in the catchall issue #6920

**Comment by bacongobbler** [comment]
The `-x` flag was removed intentionally in preparation for the Lua engine as well as Library Chart support. We weren't sure if we wanted to allow partial template execution as it may mess up how the Lua engine depends on certain templates, or that certain templates may rely on others (in terms of library charts).

Apologies for not properly communicating this out. We discussed this earlier in #5365.
